GLTFSDK/Source/MeshPrimitiveUtils.cpp

```diff
@@ -509,6 +510,123 @@ namespace
             return rawIndices;
         }
     }
+
+    template<typename T>
+    std::vector<T> ReconstructTriangleStripIndexing(const T* indices, size_t indexCount)
+    {
+        if (indexCount % 3 != 0)
+        {
+            throw GLTFException("Input triangulated triangle strip has non-multiple-of-3 indices.");
+        }
+
+        if (indexCount < 3)
+        {
+            throw GLTFException("Input triangulated triangle strip has fewer than 3 indices.");
+        }
+
+        std::vector<T> result;
```

Comment:
Call reserve on the vector(s) to avoid reallocations?

Reply:
Done.